web/src/containers/Login.js

```diff
@@ -62,6 +68,7 @@ class Login extends Component {
           cancelable={false}
           canSubmit={username.length && password.length}
           error={errorMessage}
+          success={hasEverLoggedOn ? 'You have securely logged out.' : null}
```

Comment:
Turns out the `<CardForm>` component already provides success and error notifications, we just have to put text in there.  🎉

Reply:
![high five](https://media3.giphy.com/media/3ohryczJ6sJxfbpPe8/source.gif)